content/docs/user-guide/gems/reference/ros2/_index.md

```diff
@@ -0,0 +1,119 @@
+Before running the following commands, make sure that the ROS 2 environment is setup. Replace \<distro\> with the ROS 2 distribution name (galactic, humble, ..) below and run the following command:
+```
+source /opt/ros/<distro>/setup.bash
+```
+
+* gazebo_msgs: `sudo apt install ros-${ROS_DISTRO}-gazebo-msgs`
+* Ackermann messages: `sudo apt install ros-${ROS_DISTRO}-ackermann-msgs`
+* Control toolbox `sudo apt install ros-${ROS_DISTRO}-control-toolbox`
+
+If a `desktop` installation of ROS 2 distro was selected, everything else should be there.
+
+Use this helpful command to install:
+
+```
+sudo apt install ros-${ROS_DISTRO}-ackermann-msgs ros-${ROS_DISTRO}-control-toolbox ros-${ROS_DISTRO}-nav-msgs ros-${ROS_DISTRO}-gazebo-msgs
+```
+
+## Features
+
+* Direct and natural support of ROS2 ecosystem:
+  * No bridges. Your simulation node will function as any other ROS2 node.
+    * This is also good for performance
+  * Easy way to include ROS2 dependencies.
+* Sensors:
+  * Sensor Component serving as a handy abstraction.
+  * Example implementations of Lidar, Camera, IMU sensors.
+    * Including a few Assets and prefabs which are ready to use. 
+* Automated handling of:
+  * Simulation time, publishing `/clock` supporting non-real time.
+  * Publishing of transformation frames (`/tf`, `/tf_static`).
+  * Validation for topic and namespace names.
+* Robot Control Component:
+  * A quick to use method of controlling your robot with Twist messages.
+  * Can be used with custom LUA scripting. 
+
+For a "feel" of these features, see an [example project](https://github.com/o3de/RobotVacuumSample) which uses this Gem to run navigation stack.
+
+<!-- ## Clone this repository
+
+Clone the gem repository from Github:
+
+```
+git clone https://github.com/RobotecAI/o3de-ros2-gem.git
+```
+
+## Building
+
+The Gem is built through building a project which uses it. Make sure to
+[source your ros2 workspace](https://docs.ros.org/en/rolling/Tutorials/Configuring-ROS2-Environment.html#source-the-setup-files)
+before building.
+
+## Adding Gem to your project
+
+To use this Gem in your project, you need to register the Gem with O3DE. First, clone this repository. Then,
+in o3de folder:
+```
+scripts/o3de.sh register --gem-path <PATH_TO_CLONED_ROS2_GEM>
+scripts/o3de.sh enable-gem -gn ROS2 -pp <PATH_TO_YOUR_PROJECT>
+```
+
+## Example project
+
+You can test O3DE ROS2 Gem with [this project](https://github.com/RobotecAI/Ros2WarehouseDemo).
+It will allow you to run robot navigation. All necessary assets are included. -->
+
+## User Guides
+
+Follow the [ROS 2 Gem User Guide](ros2-gem-user-guide.md) to understand its concepts and components.
+
+<!-- If you plan on contributing please follow the [Pull Request Publishing](docs/guides/pr_publishing.md) guide. For those using the Clion IDE we advise to follow the [Development in Clion](docs/guides/development_in_clion.md) guide. -->
```

Comment:
Development / contribution docs (Clion, PR) should be moved to the new Gem location in extras and referenced in the README for the Gem and also here as a link to the repo: "For additional developer's documentation, check the Gem docs (link)"